.codecov.yml

```diff
@@ -276,6 +274,3 @@ component_management:
       name: python3
       paths: ["python3/**", "!**/test_*.py"]
 
-    - component_id: test_cases
```

Comment:
Should we change it to python3 instead of removing it?

Reply:
Sure.